plugins/CarlaBase/CMakeLists.txt

```diff
@@ -1,11 +1,3 @@
-# For MacOS, use "OLD" RPATH install_name behavior
-# This can be changed to "NEW" safely if install_apple.sh.in
-# is updated to relink libcarlabase.dylib.  MacOS 10.8 uses
-# cmake 3.9.6, so this can be done at any time.
-IF(NOT CMAKE_VERSION VERSION_LESS 3.9)
-  CMAKE_POLICY(SET CMP0068 OLD)
-ENDIF()
-
```

Comment:
Adding this comment back in from the old PR:?

@PhysSong said:

> We now require CMake 3.9, so isn't it possible to either remove the if or switch to the `NEW` behavior?

Agreed, but this comment suggests Carla won't work otherwise.  Perhaps its superseded by the weak linking?  I'll make mental note, but I agree, it's pointless to leave in. <3

Reply:
> > This can be changed to "NEW" safely if install_apple.sh.in is updated to relink libcarlabase.dylib.
> 
> Isn't it still worth testing with weak linking turned off?

Sure, building now.

Reply:
Definitely broken:

```
Cannot load library /Users/owner/lmms/build/LMMS.app/Contents/lib/lmms/libcarlarack.so: (dlopen(/Users/owner/lmms/build/LMMS.app/Contents/lib/lmms/libcarlarack.so, 0x0005): Library not loaded: @rpath/libcarlabase.dylib
  Referenced from: <D312C838-3486-366D-8CF2-C18F7C851702> /Users/owner/lmms/build/LMMS.app/Contents/lib/lmms/libcarlarack.so
  Reason: tried: '/Users/owner/lmms/build/LMMS.app/Contents/Frameworks/libcarlabase.dylib' (no such file), '/opt/homebrew/Cellar/qt@5/5.15.10/lib/QtCore.framework/Versions/5/Frameworks/libcarlabase.dylib' (no such file), '/opt/homebrew/lib/libcarlabase.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/libcarlabase.dylib' (no such file), '/opt/homebrew/Cellar/sdl2/2.28.1/lib/libcarlabase.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/sdl2/2.28.1/lib/libcarlabase.dylib' (no such file), '/usr/local/lib/libcarlabase.dylib' (no such file), '/usr/lib/libcarlabase.dylib' (no such file, not in dyld cache))
Cannot load library /Users/owner/lmms/build/LMMS.app/Contents/lib/lmms/libcarlapatchbay.so: (dlopen(/Users/owner/lmms/build/LMMS.app/Contents/lib/lmms/libcarlapatchbay.so, 0x0005): Library not loaded: @rpath/libcarlabase.dylib
  Referenced from: <6130C9CD-8D91-3B9B-9893-DB7F81646467> /Users/owner/lmms/build/LMMS.app/Contents/lib/lmms/libcarlapatchbay.so
  Reason: tried: '/Users/owner/lmms/build/LMMS.app/Contents/Frameworks/libcarlabase.dylib' (no such file), '/opt/homebrew/Cellar/qt@5/5.15.10/lib/QtCore.framework/Versions/5/Frameworks/libcarlabase.dylib' (no such file), '/opt/homebrew/lib/libcarlabase.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/libcarlabase.dylib' (no such file), '/opt/homebrew/Cellar/sdl2/2.28.1/lib/libcarlabase.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/sdl2/2.28.1/lib/libcarlabase.dylib' (no such file), '/usr/local/lib/libcarlabase.dylib' (no such file), '/usr/lib/libcarlabase.dylib' (no such file, not in dyld cache))
```

Reply:
Mac support is just around broken.  I'm not sure how much of it is caused by this PR, but for starters, this line prevents the `install_apple.sh` script from finishing:

https://github.com/LMMS/lmms/blob/master/cmake/apple/install_apple.sh.in#L47

Reply:
Zyn is unrelated.  New bug filed to track that.  Carla is hard to troubleshoot until Zyn's fixed first.  I'll try to remember to jump on that next.

Reply:
If applicable, I will also attempt to track the CMP0068 related issues in the Zyn bug report.

Reply:
@tresf Could you try again, since Zyn is fixed on `master`?